packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -1214,21 +1223,20 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | TeamsCa
 
   /* @conditional-compile-remove(breakout-rooms) */
   public async returnFromBreakoutRoom(): Promise<void> {
-    if (!this.originCall) {
-      throw new Error('Could not return from breakout room because the origin call could not be retrieved.');
-    }
```

Comment:
I will remove private property `originCall` in a follow-up refactor PR since we can just call `assignedBreakoutRoom.returnToMainMeeting()` from the calling SDK to return to the main meeting